### PR TITLE
Levels review

### PR DESCRIPTION
**Pull request by alxs**

### CODE REVIEW

File: applications/application-template.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ To assist you in defining it, we have created a document with examples for some
 
 * **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
 * **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
-* **Total Costs:** Amount of payment in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-level) of your grant application. 
+* **Total Costs:** Requested amount in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-levels) of your grant application (e.g. 12,000 USD). This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```
Comment (alxs) [inline applications/application-template.md:125]: Also isn't this really the other way around? I.e. the level of the application depends on the amount of funding requested.

File: README.md [hunk 2]
```diff
@@ -118,38 +118,38 @@ The Operations Team takes care of legal documents, invoicing and remittances.
...
 ### Level 2 :baby_chick:
 
-- **Target:** Small Teams/Start-ups 
+- **Target:** Small teams/start-ups 
 - **Amount:** Up to $50,000
-- **Requirements:** 3 Approvals 
-- **Benefits:** All of the above + [Grant Announcement](./docs/announcement-guidelines.md), [Grant Badge](./docs/grant-badge-guidelines.md), Fast track to [Substrate
+- **Requirements:** 3 approvals 
+- **Benefits:** All of the above + [co-promotion](./docs/announcement-guidelines.md#announcement-guidelines), [Grants Program badge](./docs/grant-badge-guidelines.md), fast track to [Substrate
 Builders Program](https://www.substrate.io/builders-program/)
```
Comment (alxs) [inline README.md:145]:
```suggestion
- **Benefits:** All of the above + [co-promotion](./docs/announcement-guidelines.md#announcement-guidelines), [Grants Program badge](./docs/grant-badge-guidelines.md), fast track to [Substrate Builders Program](https://www.substrate.io/builders-program/) 
```

File: applications/application-template.md (hunk 1 above)
Comment (alxs) [reply applications/application-template.md:125]:
```suggestion
* **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```

**Comment by Noc2** [comment]
Thanks!

File: applications/application-template.md [hunk 3]
```diff
@@ -122,7 +122,7 @@ To assist you in defining it, we have created a document with examples for some
 
 * **Total Estimated Duration:** Duration of the whole project (e.g. 2 months)
 * **Full-Time Equivalent (FTE):**  Average number of full-time employees working on the project throughout its duration (see [Wikipedia](https://en.wikipedia.org/wiki/Full-time_equivalent), e.g. 2 FTE)
-* **Total Costs:** Amount of payment in USD for the whole project. The maximum amount of funding depends on the [level](../README.md#level_slider-level) of your grant application. 
+* **Total Costs:** Requested amount in USD for the whole project (e.g. 12,000 USD). Note that the acceptance criteria and additional benefits vary depending on the [level](../README.md#level_slider-levels) of funding requested. This and the costs for each milestone need to be provided in USD; if the grant is paid out in Bitcoin, the amount will be calculated according to the exchange rate at the time of payment.
```
Comment (alxs) [inline applications/application-template.md:125]: I missed whether or not we want to allow teams to apply for a higher level than their funding requires, in my opinion this is additional complexity that we don't really need. If a team want the benefits of level 2 I think it's fair to ask that they deliver something at least worth $10k, and if they really want VC introductions they can just ask.

**Pull request merged**